src/app/query.rs

```diff
@@ -629,6 +630,10 @@ pub enum PrefixType {
     State,
     User,
     Time,
+    #[cfg(feature = "gpu")]
+    PGpu,
+    //#[cfg(feature = "gpu")]
```

Comment:
Should this be uncommented?

Reply:
it is used in the above match arm. I can look into enabling the feature gate by pulling the above match arm branch into a function.